HIP/hip-904.md

```diff
@@ -7,11 +7,12 @@ requested-by: DEXs, Wallets, External Web3 Users. Notably Reality+, Liithos, Dro
 type: Standards Track
 category: Core
 needs-council-approval: Yes
-status: Accepted
+status: Final
+release: v0.116.1
```

Comment:
HIPs don't support versions for multiple components so here we would be the services version